web/docs/_static/openapi.yaml

```diff
@@ -492,6 +512,10 @@ components:
             curious how that will be for me.
         last_signal:
           $ref: '#/components/schemas/signal'
+        avatar:
+          type: string
+          description: User's gravatar
+          example: 
```

Comment:
It would be good to provide an example here.

Reply:
Done!